more/free-programming-interactive-tutorials-en.md

```diff
@@ -196,6 +196,7 @@
 ### Java
 
 * [CodingBat code practice](http://codingbat.com/java)
+* [Free API World Cup 2022](https://github.com/raminmr/free-api-worldcup2022) - Raminmr
```

Comment:
hey @cristhiannrodrigues, can you explain what this resource is? I just see one README file